packages/theming/atlas/content/index.html

```diff
@@ -7,10 +7,10 @@
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="apple-mobile-web-app-capable" content="yes" />
 
-        {{themecss}} {{appicons}}
-
-        <!-- STARTUP IMAGES -->
-        <!-- <link href="resources/ios/startup/apple-touch-startup-image-1496x2048.png" media="(device-width: 768px) and (device-height: 1024px) and (-webkit-device-pixel-ratio: 2) and (orientation: landscape)" rel="apple-touch-startup-image"> -->
+        {{themecss}}
+        {{appicons}}
```

Comment:
extra spacing :(

Reply:
Yes that is intentional as otherwise multiple things will end up on the same line in the generated html.